eng/Version.Details.xml

```diff
@@ -1,12 +1,8 @@
 <Dependencies>
   <ProductDependencies>
-    <Dependency Name="Microsoft.Diagnostics.Monitoring" Version="5.0.328102">
+    <Dependency Name="Microsoft.Diagnostics.Monitoring" Version="6.0.0-preview.22609.1">
       <Uri>https://github.com/dotnet/diagnostics</Uri>
-      <Sha>ab657cc109f93e48d52f6b12cd7f136bb2d0b311</Sha>
-    </Dependency>
-    <Dependency Name="Microsoft.Diagnostics.Monitoring.EventPipe" Version="5.0.328102">
```

Comment:
Was this removed intentionally?

Reply:
It was, but I can add it back so that development versions are used independently.

Reply:
I've separated the versions for each of the libraries so that they are specified independently for darc updates and identically for dependabot updates.